include/openthread/instance.h

```diff
@@ -52,7 +52,7 @@ extern "C" {
  *
  * @note This number versions both OpenThread platform and user APIs.
  */
-#define OPENTHREAD_API_VERSION (470)
+#define OPENTHREAD_API_VERSION (471)
```

Comment:
No need to increment this.

Reply:
Yes, I understand, this is not public API.

Reply:
Fixed.